api/envoy/config/core/v3/protocol.proto

```diff
@@ -55,6 +55,9 @@ message QuicProtocolOptions {
   // window size now, so it's also the minimum.
   google.protobuf.UInt32Value initial_connection_window_size = 3
       [(validate.rules).uint32 = {lte: 25165824 gte: 1}];
+
+  // If true, quic client connection will try migrating to a different port when network path is degrading. This has no effect on server sessions.
+  bool migrate_port_on_path_degrading = 4;
```

Comment:
This should probably use `google.protobuf.BoolValue`, as per https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md#feature-enablement,

Reply:
done.